client/src/main/java/io/pravega/client/control/impl/Controller.java

```diff
@@ -364,6 +364,14 @@
      */
     CompletableFuture<Transaction.Status> checkTransactionStatus(final Stream stream, final UUID txId);
 
+    /**
+     * Method to get transaction list in completed state(COMMITTED, ABORTED).
+     *
+     * @param stream Stream Name
+     * @return list of transaction ids
+     */
+    CompletableFuture<List<UUID>> listCompletedTxns(final Stream stream);
```

Comment:
Better wo use `listCompletedTransactions` instead of `listCompletedTxns`

Reply:
Done as suggested.